lib/que/job.rb

```diff
@@ -29,21 +29,18 @@ class Job
 
     SQL[:bulk_insert_jobs] =
       %{
-        WITH args_and_kwargs as (
-          SELECT * from json_to_recordset(coalesce($5, '[{args:{},kwargs:{}}]')::json) as x(args jsonb, kwargs jsonb)
-        )
         INSERT INTO public.que_jobs
         (queue, priority, run_at, job_class, args, kwargs, data, job_schema_version)
         SELECT
-          coalesce($1, 'default')::text,
-          coalesce($2, 100)::smallint,
-          coalesce($3, now())::timestamptz,
-          $4::text,
-          args_and_kwargs.args,
-          args_and_kwargs.kwargs,
-          coalesce($6, '{}')::jsonb,
+          coalesce(queue, 'default')::text,
```

Comment:
I didn't know you could refer to these by name! Excellent

Reply:
Yeah, this is only possible because it's using [json_populate_recordset](https://www.postgresql.org/docs/current/functions-json.html#FUNCTIONS-JSON-PROCESSING-TABLE). A nice cleanup IMO.